src/mixins/colorable.ts

```diff
@@ -1,17 +1,54 @@
 import Vue from 'vue'
 
+export interface ClassesObject {
+  [key: string]: boolean
+}
+
+export type ColorString = string | undefined | null | false
+
+function addColor (
+  classes: ClassesObject = {},
+  color?: ColorString
+): ClassesObject {
+  const obj = { ...classes }
+
+  if (color) obj[color] = true
```

Comment:
we could just return `classes` if `color` is falsy

Reply:
alternatively **add**Color function could **always** add color, but it should be called only if color is not falsy:
`return color ? addColor(classes, color) : classes`

that would simplify `addColor` function, so that imho it's not needed anymore:

```
function addBackgroundColorClassChecks() {
  return color ? {
    ..classes,
    [color]: true
  } : classes
}
function addTextColorClassChecks() {
 if (!color) return classes
 ...
 return {
    ...classes,
    color
 }
}
```